### ISSUE TITLE
Virtual method not being called in wx.Sizer in 4.1 dev version

### ISSUE THREAD

**Issue by efahl** [wxWidgets/Phoenix#1356]
Win 10 64, standard CPython 3.7.4

Downloaded and pip installed wxPython-4.1.0a1.dev4340+af214853-cp37-cp37m-win_amd64.whl, found that the method overrides are not being called on wx.Sizer (well, at least RecalcSizes, didn't check others).

```python
import wx

class Sizer(wx.Sizer):
    def CalcMin(self):
        return wx.Size(0,0)
    def RecalcSizes(self):
        print('yeah, got here')

wx.App()
f = wx.Frame()
f.SetSizer(Sizer())
f.Layout()
```

With wxPython 4.0.6, 
```
> python sizer.py
yeah, got here
```

With the dev 4.1,
```
> python sizer.py
Traceback (most recent call last):
  File "sizer.py", line 12, in <module>
    f.Layout()
wx._core.wxAssertionError: C++ assertion ""Assert failure"" failed at ..\..\src\common\sizer.cpp(994) in wxSizer::RecalcSizes(): Must be overridden if RepositionChildren() is not
```

**Comment by RobinD42** [comment]
There have been some changes here in wxWidgets. `RecalcSizes` has been removed from the public API and replaced with `RepositionChildren`. `RecalcSizes` still exists in the C++ class for compatibility, and the default `RepositionChildren`, if not overridden, will call `RecalcSizes`.

Since it has been removed from the documented API I wrote the code to add the `RecalcSizes` back into the class definition for wxPython, but it looks like I forgot the one extra line to actually inject the method into the class. (Oops!)

I'll get it fixed for the next build.
